lib/grpc/server/supervisor.ex

```diff
@@ -48,7 +48,8 @@ defmodule GRPC.Server.Supervisor do
     check_deps_version()
 
     children =
-      if Application.get_env(:grpc, :start_server, false) do
+      if (Keyword.has_key?(opts, :start_server) and opts[:start_server]) or
+           Application.get_env(:grpc, :start_server) do
```

Comment:
❤️

Reply:
Just caught a slight bug on the logic here haha!

The has_key check is slightly off